include/line_numbers.h

```diff
@@ -0,0 +1,52 @@
+#ifndef LINE_NUMBERS_H
+#define LINE_NUMBERS_H
+
+// The macros in this file serve to reduce clutter from version differences that are only due to line numbers being different.
+
+/**
+ * Macro that defines line numbers by version.
+ * This macro is not meant to be used externally, see wrappers below.
+ */
+#define LN(ntsc_1_0, ntsc_1_1, ntsc_1_2, pal_1_0, pal_1_1, gc_jp, gc_jp_mq, gc_us, gc_us_mq, gc_eu, gc_eu_mq, gc_jp_ce, debug) \
+    (OOT_VERSION == NTSC_1_0) ? ntsc_1_0 : \
+    (OOT_VERSION == NTSC_1_1) ? ntsc_1_1 : \
+    (OOT_VERSION == NTSC_1_2) ? ntsc_1_2 : \
+    (OOT_VERSION == PAL_1_0)  ? pal_1_0  : \
+    (OOT_VERSION == PAL_1_1)  ? pal_1_1  : \
+    (OOT_VERSION == GC_JP)    ? gc_jp    : \
+    (OOT_VERSION == GC_JP_MQ) ? gc_jp_mq : \
+    (OOT_VERSION == GC_US)    ? gc_us    : \
+    (OOT_VERSION == GC_US_MQ) ? gc_us_mq : \
+    (OOT_VERSION == GC_EU)    ? gc_eu    : \
+    (OOT_VERSION == GC_EU_MQ) ? gc_eu_mq : \
+    (OOT_VERSION == GC_JP_CE) ? gc_jp_ce : debug
```

Comment:
I'm slightly concerned that this will could affect matching somehow, and that it would be better to do

```c
#if OOT_VERSION == NTSC_1_0
#define LN(...) (ntsc_1_0)
#elif OOT_VERSION == NTSC_1_1
...
```

maybe not a bit deal though

Reply:
I like the way the ternary looks more personally but dont have a super strong preference. I think if it does effect match at some point we can switch?
Or I can just do it now if there is a stronger preference toward that.

Reply:
I'm kind of afraid someone (e.g. me) will waste a bunch of time trying to get something to match (or to find a less fake match, or whatever) without realizing the `LN` macro is the culprit. So I think no ternaries is the "safer" choice even though it doesn't matter right now.

Although, no ternaries it would be a lot uglier, and not too many people care about finding new matches anyway. I think I don't really care, just thought I'd throw the idea out there

Reply:
One downside with this method is multiple macro definitions means multiple arguments that will need to be updated as versions are added. Instead of just doing it in one place.

Im fine with this if you are, just wanted to put it out there. If yes I will make the change.

Reply:
yeah that's true, but I'm fine with it because it'll still only require changing one "code block" and adding the new param can be done with a find-and-replace so it's not too bad

Reply:
alright, changes made